### PR TITLE
Changes#1

### PR DESCRIPTION
**Pull request by kamilczysko**
I have made some changes in code according to your hints.

### CODE REVIEW

File: src/BasicStringValue.java [hunk 1]
```diff
@@ -1,30 +1,32 @@
 import java.util.Optional;
 
 public class BasicStringValue extends Value {
+    private String actualValue = null;
 
-    private String value = null;
-
-    public BasicStringValue(String value) {setValue(value);}
+    public BasicStringValue(String mValue) {
```
Comment (XMks894) [inline src/BasicStringValue.java]: what do you think about mValue like an argument in BasicStringValue? in some Java nomenclature "m" is the synonime of field of class. Are you sure that is a good idea to rename to mValue?

File: src/ComplexValue.java [hunk 2]
```diff
@@ -4,15 +4,14 @@
 import java.util.Stack;
 
 public class ComplexValue extends Value {
-
     private Stack<Value> parametersStack;
 
     public void setParametersStack(){
         parametersStack = new Stack();
-        List<Value> vals = new LinkedList<>(getParameters());
+        List<Value> listOfValues = new LinkedList<>(getParameters());
```
Comment (XMks894) [inline src/ComplexValue.java]: i think that rename from vals to list will be enough in that case... what do you think?

File: src/BasicStringValue.java (hunk 1 above)
Comment (kamilczysko) [reply src/BasicStringValue.java]: I think that names of variables or methods should describe their role in class. 
For me prefix 'm' makes variables names more enigmatic.

File: src/ComplexValue.java (hunk 2 above)
Comment (kamilczysko) [reply src/ComplexValue.java]: Yes, you are right. This is local variable and it's not necessary and making such complex names is redundant.

**Pull request merged**